Packs/NucleonCyber/Integrations/NucleonCyberFeed/NucleonCyberFeed.py

```diff
@@ -0,0 +1,761 @@
+        for key, value in item.items():
+            raw_data.update({key: value})
+
+        indicator_obj = {
+            # The indicator value.
+            'value': value_,
+            'type': type_,
+            # The indicator type as defined in Cortex XSOAR.
+            # One can use the FeedIndicatorType class under CommonServerPython to populate this field.
+            # 'exp':exp_,
+            # "port":port_,
+            # The name of the service supplying this feed.
+            'service': 'NucleonCyberFeed',
+            # A dictionary that maps values to existing indicator fields defined in Cortex XSOAR.
+            # One can use this section in order to map custom indicator fields previously defined
+            # in Cortex XSOAR to their values.
+            'fields': {
+            },
+            # A dictionary of the raw data returned from the feed source about the indicator.
+            'rawJSON': raw_data
+        }
+        indicators.append(indicator_obj)
+
+    return indicators
+
+
+def get_indicators_command(client: Client,
+                           params: Dict[str, str],
+                           args: Dict[str, str]
+                           ) -> CommandResults:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+        args: demisto.args()
+    Returns:
+        Outputs.
+        IP indicators
+    """
+    limit = int(args.get('limit', '10'))
+    tlp_color = params.get('tlp_color')
+    feed_tags = argToList(params.get('feedTags', ''))
+    indicators = fetch_indicators(client, tlp_color, feed_tags, limit)
+    human_readable = tableToMarkdown('IP indicators from NucleonCyberFeed:', indicators,
+                                     headers=['value', 'type', 'exp'], headerTransform=string_to_table_header, removeNull=True)
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='NucleonCyber.Indicators',
+        outputs_key_field='',
+        raw_response=indicators,
+        # outputs={},
+        outputs=indicators,
+    )
+
+
+def fetch_indicators_command(client: Client, params: Dict[str, str]) -> List:
+    """Wrapper for fetching indicators from the feed to the Indicators tab.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+    Returns:
+        All indicators type (ip,url,hash(file)).
+    """
+    feed_tags = argToList(params.get('feedTags', ''))
+    tlp_color = params.get('tlp_color')
+    ips = fetch_indicators(client, tlp_color, feed_tags)
+    urls = fetch_urls(client)
+    hashes = fetch_hashes(client)
+    return [ips, urls, hashes]
+
+
+"""
+"""
+
+
+def get_hashes_command(client: Client,
+                       params: Dict[str, str],
+                       args: Dict[str, Any]
+                       ) -> CommandResults:
```

Comment:
Please remove params argument as it is not used in function
```suggestion
def get_hashes_command(client: Client,
                       args: Dict[str, Any]
                       ) -> CommandResults:
```

Reply:
removed